### PR TITLE
Commerce 4497 discount level labels

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -500,6 +500,10 @@ dimensions=Dimensions
 discount-amount=Discount Amount
 discount-application-strategy=Discount Application Strategy
 discount-level=Discount Level
+discount-level-1=L1
+discount-level-2=L2
+discount-level-3=L3
+discount-level-4=L4
```
Comment (dsitu) [inline modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties:506]: @andrea-ale-sbarra can we make it like this so the keys and values match?
```suggestion
l1=L1
l2=L2
l3=L3
l4=L4
```

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by dsitu** [comment]
Closing resent here https://github.com/liferay-commerce/liferay-portal/pull/189.

**Pull request closed without merging**